x-pack/plugins/lens/public/metric_visualization_plugin/metric_config_panel.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ReactWrapper } from 'enzyme';
+import { mountWithIntl as mount } from 'test_utils/enzyme_helpers';
+import { MetricConfigPanel } from './metric_config_panel';
+import { DatasourcePublicAPI, DatasourceDimensionPanelProps, Operation } from '../types';
+import { State } from './types';
+import { NativeRendererProps } from '../native_renderer';
+
+describe('MetricConfigPanel', () => {
+  const dragDropContext = { dragging: undefined, setDragging: jest.fn() };
+
+  function mockDatasource(): DatasourcePublicAPI {
+    return {
+      duplicateColumn: () => [],
+      getOperationForColumnId: () => null,
+      generateColumnId: () => 'TESTID',
+      getTableSpec: () => [],
+      moveColumnTo: () => {},
+      removeColumnInTableSpec: () => [],
+      renderDimensionPanel: () => {},
+    };
+  }
+
+  function testState(): State {
+    return {
+      title: 'Test Metric',
+      accessor: 'foo',
+    };
+  }
+
+  function testSubj(component: ReactWrapper<unknown>, subj: string) {
+    return component
+      .find(`[data-test-subj="${subj}"]`)
+      .first()
+      .props();
+  }
+
+  test('allows editing the chart title', () => {
```

Comment:
Do we need the title with the saved object PR?

Reply:
No. This'll change when I merge this with that branch.